src/EthernetServer.h

```diff
@@ -14,7 +14,8 @@ public Server {
   
   void accept(void);
 public:
-  EthernetServer(uint16_t);
+  EthernetServer();
+  EthernetServer(uint16_t port);
```

Comment:
```suggestion
  EthernetServer(uint16_t port = 80);
```
Like this only need to define one